src/test/java/games/strategy/engine/lobby/server/login/RsaAuthenticatorTest.java

```diff
@@ -3,24 +3,25 @@
 import static java.util.Collections.singletonMap;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
 
-import java.security.PrivateKey;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.function.Function;
 
+import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mock;
-import org.mockito.junit.MockitoJUnitRunner;
 
-import com.google.common.cache.Cache;
-import com.google.common.cache.CacheBuilder;
+import games.strategy.security.TestSecurityUtils;
 
-@RunWith(MockitoJUnitRunner.StrictStubs.class)
-public class RsaAuthenticatorTest {
+public final class RsaAuthenticatorTest {
+  private RsaAuthenticator rsaAuthenticator;
 
-  @Mock
-  private PrivateKey mockPrivateKey;
+  @Before
+  public void setUp() throws Exception {
+    rsaAuthenticator = new RsaAuthenticator(TestSecurityUtils.loadRsaKeyPair());
+  }
```

Comment:
I see you added a lot of logic in the security util class...
Wouldn't it just be easier to use the `@BeforeClass` annotation and use a static method + field instead?

Reply:
I actually did do that initially.  I then moved `loadRsaKeyPair()` (and its relatives) to a separate class because

a) there was enough code there to justify separation of concerns, and
b) multiple test classes (`RsaAuthenticatorTest` and `LobbyLoginValidatorTests`) require access to this method, and I didn't want to duplicate the code in multiple locations.